tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary.feature

```diff
@@ -28,31 +33,71 @@ Scenario: 1. Displaying the order summary, where the order sections are complete
         | additional-services  | incomplete |
         | funding-source       | incomplete |
 
+@7221
+Scenario: 2. Displaying the order summary, where the ordering party section is complete
+    When the user makes a request to retrieve the order summary with the ID C000014-02
+    Then a response with status code 200 is returned
+    And the order summary is returned with the following values
+        | OrderId    | OrganisationId                       | Description   |
+        | C000014-02 | 4af62b99-638c-4247-875e-965239cd0c48 | A Description |
+    And the order Summary Sections have the following values
+        | Id                  | Status     |
+        | description         | complete   |
+        | ordering-party      | complete   |
+        | supplier            | incomplete |
+        | commencement-date   | incomplete |
+        | associated-services | incomplete |
+        | service-recipients  | incomplete |
+        | catalogue-solutions | incomplete |
+        | additional-services | incomplete |
+        | funding-source      | incomplete |
+    And the order with orderId C000014-02 has a primary contact
+
+@7221
+Scenario: 3. Displaying the order summary, where the ordering party section is incomplete

```

Comment:
Is this just the same test as above? No1?

Reply:
Explicitly called  out the negative case and proved the order was in a state that would not set the section as complete.  But i can delete it